src/frontend/handlers.go

```diff
@@ -79,8 +80,20 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
-	//get env and render correct platform banner.
-	var env = os.Getenv("ENV_PLATFORM")
+	// Set ENV_PLATFORM (checks for GCP. otherwise uses env value.)
+	var env = "local"
+	addrs, err := net.LookupHost("metadata.google.internal.")
```

Comment:
I wonder if there's a way to do this without hard-coding a conditional for it. How is for example the Azure case handled? Do we always expect the user / something to set the env correctly, which is why we don't have to handle it here? Does it make sense to do something similar for GCP, or are we doing it more explicitly to remove a manual step somewhere else?

Reply:
Good point, I added a validity check for the env var to make sure it's a flag we have CSS for. (the flags each have different colors.)  This PR is really just designed to address the request to autodetect GCP. not sure if there's a way to autodetect azure + aws in the same way.